common/persistence/class.PhpFileDriver.php

```diff
@@ -202,8 +202,11 @@ private function writeFile($id, $value, $preWriteValueProcessor = null)
     public function get($id)
     {
         $entry = $this->readFile($id);
-        if ($entry != false && $this->isTtlMode()) {
-            $entry = (is_null($entry[static::ENTRY_EXPIRATION]) || $entry[static::ENTRY_EXPIRATION] > $this->getTime())
+
+        if ($entry !== false && $this->isTtlMode()) {
```

Comment:
```suggestion
        if ($entry !== false && $this->isTtlMode()) {
```
@jbout Please, confirm that I can change this behavior: before it worked like this: when file has `return null` - driver return `null ` as a result, but now in such situation it returns 'false'